### DISCUSSION TITLE
想问一下是否有关于引擎性能的 Benchmark？以及底层架构的说明，为什么性能比其他产品好。

### DISCUSSION THREAD

**Discussion by 6174** [galacean/discussion#40] [Engine]
从官网看到产品的定位是 “移动优先的高性能 Web 互动引擎”， 但是高性能似乎没有更进一步的说明，想进一步了解一下。  

> 仅个人使用者的视角反馈，我的第一想法是想要知道 WHY - 为什么选择 Galacean，高性能、移动端、API 易用性是吸引用我的点，但是翻了所有的文档似乎没有进一步说明，所以想进一步问问有没有相关 Benchmark。

**Comment by 6174** [comment]
另外补充一下，官网上写的三大特别 “组件化设计” 、 “移动优先”、“前端友好” 似乎都不那么明确感受到差异化价值， 建议可以增加更详细的文档说明链接，和其他产品进行对比一下可能更直观。   

特别是第一的 “组件化设计” 看上去是一个很棒的底层架构设计，但没有案例不太能感受到。

**Comment by eyworldwide** [comment]
很好的建议！👍

> 但是高性能似乎没有更进一步的说明，想进一步了解一下。

高性能体现在引擎的每个功能点中，具体的优化因为并不影响到引擎 API 的使用，所以没有明显体现在官网内容中，如果有兴趣可以点开 https://www.yuque.com/galacean-engine/blog 博客中看一看，里面有一些文章讲到性能优化。另外，我们考虑到对行业其他同类产品的尊重，所以官方暂不提供对比数据。

不过，官网示例中提供了一些 Benchmark，比如 https://galacean.antgroup.com/#/examples/latest/benchmark-animation

有兴趣的话，您可以使用同类引擎做同一示例在帧率、内存方面的对比测试。


> 另外补充一下，官网上写的三大特别 “组件化设计” 、 “移动优先”、“前端友好” 似乎都不那么明确感受到差异化价值， 建议可以增加更详细的文档说明链接，和其他产品进行对比一下可能更直观。

这方面的文档确实不太完善，我们会专门补充完善。
